neuralop/models/fno.py

```diff
@@ -152,6 +164,17 @@ def __init__(
         self.preactivation = preactivation
         self.fno_block_precision = fno_block_precision
 
+        if spatial_pos_embed == "grid":
```

Comment:
spatial_pos_embed seems redundant, why not just `positional_embedding`?

Reply:
In case we add embedding in Fourier space, to avoid ambiguity

Reply:
Let's call it positional_embedding (or encoding) it will be clearer. Fourier space embedding is still being tested and can be named differently.

Reply:
Sounds good